agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/configuration/Configuration.java

```diff
@@ -1230,13 +1230,90 @@ public void validate() {
     }
   }
 
+  public enum RequestFilterType {
+    REGEX
+  }
+
+  public static class RequestFilter {
+    public RequestFilterType type;
+    public String value;
+  }
+
+  public static class RequestAggregationConfig {
+
+    // Threshold in ms over which a span will consider to be a breach
+    // Used by the breach ratio aggregation
+    public int thresholdMs = 5000;
+
+    // Minimum number of samples that must have been collected in order for the aggregation to
+    // produce data. Avoids volatile aggregation output on small sample sizes.
+    public int minimumSamples = 0;
+  }
+
+  public enum RequestAggregationType {
+    BREACH_RATIO
```

Comment:
this has been our convention for configuration enum mappings:

```suggestion
    @JsonProperty("breach-ratio")
    BREACH_RATIO
```

Reply:
I also applied this to `name-regex` and `greater-than` correct?